hw2/05-form.js

```diff
@@ -1 +1,24 @@
 // Add your code here
+const form = document.querySelector('form');
+
+function handleSubmit(e) {
+  e.preventDefault();
+  const elementNames = ['full-name', 'email', 'registration-status', 'comments'];
+  const formElements = form.elements;
+  for (let i = 0; i < elementNames.length; i++) {
+    const element = formElements[elementNames[i]];
+    console.log(`${element.name}: ${element.value}`);
+  }
+
+  if (formElements.programmingLanguages) {
```

Comment:
A possible rework would be to only output the name of the course if the course is selected. You could create a console.group for the courses and then include the name of all the courses selected in the group.